torch/onnx/_internal/exporters/dynamo_optimize.py

```diff
@@ -0,0 +1,48 @@
+import copy
+from typing import Optional
+
+import torch._dynamo
+from torch.onnx._internal.exporter import ExportOutput
+from torch.onnx._internal.exporters.fx_base import FXGraphModuleExporter
+
+
+class DynamoOptimizeExporter(FXGraphModuleExporter):
```

Comment:
Same as above (new stuff not discussed on v1 design proposal which deviates from the complete design)